tools/acrntrace/README.rst

```diff
@@ -12,6 +12,71 @@ A ``scripts`` directory includes scripts to analyze the trace data.
 Usage
 *****
 
+The ``acrntrace`` tool runs on the Service OS (SOS) to capture trace data and
+output to trace file under ``/tmp/acrntrace`` with raw (binary) data format.
+
+Options:
+
+-h			print this message
+-t period		specify polling interval in milliseconds [1-999]
+-c			clear the buffered old data
+
+The ``acrntrace_format.py`` is a offline tool for parsing trace data (as output
+by acrntrace) to human-readable formats based on given format.
+
+Here's an explanation of the tool's parameters:
+
+.. code-block:: none
+
+   acrntrace_format.py [options] [formats] [trace_data]
+
+Options:
+
+-h    print this message
+
+  *formats* file specifies the rules to reformat the *trace_data* collected by
```

Comment:
Remove the indent (leading two spaces) on all this text (lines 37-53), except for the ``event_id  text_format_string`` on line 41).  (As written, it looks like it's a continuation of the text for the -h option.)

Reply:
Will do. Thanks David.